iati/about/models.py

```diff
@@ -1,8 +1,145 @@
 from django.db import models
 
-from wagtail.core.models import Page
+from wagtail.admin.edit_handlers import InlinePanel
+from wagtail.core.blocks import CharBlock, StreamBlock, StructBlock, TextBlock
+from wagtail.core.fields import StreamField
+from wagtail.core.models import Orderable
+from wagtail.documents.edit_handlers import DocumentChooserPanel
+from wagtail.images.blocks import ImageChooserBlock
+from wagtail.images.edit_handlers import ImageChooserPanel
 
-class AboutPage(Page):
-    pass
-    # parent_page_types = ['home.HomePage']
-    # subpage_types = ['home.AboutSubpage']
+from modelcluster.fields import ParentalKey
+from home.models import AbstractContentPage, AbstractIndexPage, IATIStreamBlock
+
+
+class AboutPage(AbstractContentPage):
+    """A model for the About landing page."""
+
+    parent_page_types = ['home.HomePage']
+    subpage_types = ['about.AboutSubPage', 'about.CaseStudyIndexPage', 'about.HistoryPage', 'about.PeoplePage']
+
+
+class AboutSubPage(AbstractContentPage):
+    """A model for generic About subpages."""
+
+    subpage_types = ['about.AboutSubPage']
+
+    multilingual_field_panels = [
+        InlinePanel('about_sub_page_documents', label='About subpage attachments'),
+    ]
+
+
+class AboutSubPageDocument(Orderable):
+    """A model for About sub page documents."""
+
+    page = ParentalKey(AboutSubPage, related_name='about_sub_page_documents')
+    document = models.ForeignKey(
+        'wagtaildocs.Document',
+        on_delete=models.CASCADE,
+        related_name='+'
+    )
+
+    panels = [
+        DocumentChooserPanel('document'),
+    ]
+
+
+class CaseStudyIndexPage(AbstractIndexPage):
+    """A model for the Case Studies Index page."""
+
+    subpage_types = ['about.CaseStudyPage']
+
+    @property
+    def case_studies(self):
+        """Get all CaseStudyPage objects that have been published."""
+        case_studies = CaseStudyPage.objects.child_of(self).live()
+        return case_studies
+
+    def get_context(self, request):
+        """Overwrite the default wagtail get_context function to allow for pagination.
+
+        Use the functions built into the abstract index page class to apply pagination, limiting the results to 3 per page.
+
+        """
+        children = self.case_studies
+        paginated_children = self.paginate(request, children, 3)
+        context = super(CaseStudyIndexPage, self).get_context(request)
+        context['case_studies'] = paginated_children
+        return context
+
+
+class CaseStudyPage(AbstractContentPage):
+    """A model for Case Study pages."""
+
+    parent_page_types = ['about.CaseStudyIndexPage']
+    subpage_types = []
+
+    feed_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
+
+    multilingual_field_panels = [
+        ImageChooserPanel('feed_image'),
```

Comment:
Suggest add help text, to show the user help text to show where this image is displayed.